Source/Terminals.Plugins.Web/WebOptions.cs

```diff
@@ -12,12 +12,26 @@ namespace Terminals.Data
     public class WebOptions : ProtocolOptions, IRelativeUrlProvider
     {
         public string RelativeUrl { get; set; }
+        public string UsernameID { get; set; }
+        public string PasswordID { get; set; }
+        public string OptionalID { get; set; }
+        public string OptionalValue { get; set; }
+        public string SubmitID { get; set; }
+        public bool EnableHTMLAuth { get; set; }

```

Comment:
Why is the EnableHtmlAuth separated from EnableFormsAuth, what is the difference. I would expect one dropdown to select authentication method. Or is the purpose as fallback authentication?

Reply:
Added both to be able to keep both enabled by default, to ease for users (Not having to chose by default).